src/pybind/mgr/rook/module.py

```diff
@@ -381,7 +381,30 @@ def remove_stateless_service(self, service_type, service_id):
             lambda: self.rook_cluster.rm_service(service_type, service_id), None,
             "Removing {0} services for {1}".format(service_type, service_id))
 
-    def create_osds(self, spec):
+    def create_osds(self, drive_group):
```

Comment:
Can we keep the API as it is?

Reply:
Of course, we can keep it without modifications. But taking into account that probably rook and ansible was the first two orchestrators with the create_osd feature implemented, i though that it was a good moment to make the specification more explicit and clear, if you check the orchestrator.py module you will see that i also have changed in it.

Apart of that, probably we will need to improve documentation around this point:
http://docs.ceph.com/docs/master/mgr/orchestrator_modules/#osd-management
 
If everybody agree with the change "s/spec/drive_group" , i can modify also the doc. If not agree i can revert the change, of course.